src/development/LocalTest/Services/Register/Implementation/OrganizationsWrapper.cs

```diff
@@ -11,17 +12,36 @@ namespace LocalTest.Services.Register.Implementation
     public class OrganizationsWrapper : IOrganizations
     {
         private readonly TestDataService _testDataService;
+        private readonly AltinnOrgsClient _orgsClient;
 
-        public OrganizationsWrapper(TestDataService testDataService)
+        public OrganizationsWrapper(TestDataService testDataService, AltinnOrgsClient orgsClient)
         {
             _testDataService = testDataService;
+            _orgsClient = orgsClient;
         }
 
         /// <inheritdoc />
         public async Task<Organization?> GetOrganization(string orgNr)
         {
             var data = await _testDataService.GetTestData();
-            return data.Register.Org.TryGetValue(orgNr, out var value) ? value : null;
+
+            if (data.Register.Org.TryGetValue(orgNr, out var value))
+            {
+                return value;
+            }
+
+            // Make lookup work for all the orgs that has apps in altinn.
```

Comment:
Which scenario will this be relevant since the first parts are organizations and the other parts are service owners?

Reply:
See https://github.com/Altinn/app-lib-dotnet/pull/176. The serviceOwner gets it's orgNumber in the `instance.LastChangedBy` field when instantiating, and this change is required for the lookup to succeed.

Reply:
```suggestion
            // In some cases the service owners org number might be used and need
            // to be looked up to show the name instead. It happens at least in the
            // {org}/{app}/instances/{instanceOwnerPartyId}/active endpoint
            // when LastChangedBy, might be the org number of a service owner
            // and we use the real service owner in LocalTest because the service owner code
            // is hard coded into policy.xml
```